src/services/filterDestination.js

```diff
@@ -0,0 +1,26 @@
+export function filterDestination(redirectTo, purposeParams, userID, redirectID){
```

Comment:
Wasn't this supposed to be called `redirectToDestination`? 

Add a docstring at the tope of the function. Something like "Based on params, sends to correct destination"

Reply:
Yes I was also confused
What I think that it should be just like the file `validation.js`

In `validation.js`, there will be code that will handle different kinds of validations. One type of validation is to validate SRN. There could be other type of validations as well, and all of them are defined as functions that can be exported.

Similarly, here, there should be a file `redirection.js` and it will handle all different kinds of redirection. One type of redirection is redirection to Plio, another type is redirection to Gmeet, similarly with Zoom etc
And those redirection logic for each should be in separate functions which can be exported.